bedrock/mozorg/templates/mozorg/home/includes/nav.html

```diff
@@ -0,0 +1,38 @@
+<header class="section masthead" id="masthead">
+  <div class="content">
+    <h1 class="masthead-logo">
+    {% if is_home %}
+      <img src="{{ static('img/home/2016/mozilla-wordmark.svg') }}" alt="Mozilla">
+    {% else %}
+      <a href="{{ url('mozorg.home') }}">
+        <img src="{{ static('img/home/2016/mozilla-wordmark.svg') }}" alt="Mozilla">
```

Comment:
Instead of repeat the `src` again here would it be worth using a jinja variable via `set`?